src/ddl/migrations/covid_hosp_state_split_tables.sql

```diff
@@ -0,0 +1,148 @@
+-- 1. Add new state_daily table mirroring state_timeseries table
+
+CREATE TABLE `covid_hosp_state_daily` (
```

Comment:
since this essentially duplicates `covid_hosp_state_timeseries`, you can use a shortcut to create it...  see [v4_schema.sql](https://github.com/cmu-delphi/delphi-epidata/blob/4e46532388de61e30ed010334c4ad0f0f11f4749/src/ddl/v4_schema.sql#L54-L59) for an example.